sig-instrumentation/annual-report-2022.md

```diff
@@ -0,0 +1,150 @@
+# 2021 Annual Report: SIG Instrumentation
+
+## Current initiatives
+
+1. What work did the SIG do this year that should be highlighted?
+
+   - Formed WG Structured Logging. Successfully migrated multiple components to structured logs and graduated feature to beta.
+   - Added tracing support to the Kubernetes API server and began work on Kubelet tracing.
+   - Graduated the metrics stability framework.
+
+2. What initiatives are you working on that aren't being tracked in KEPs?
+
+   - Bi-weekly triage meeting
+   - Subprojects (see below)
+
+3. KEP work in 2021 (1.x, 1.y, 1.z):
+
+<!--
+In future, this will be generated from kubernetes/enhancements kep.yaml files
+1. with SIG as owning-sig or in participating-sigs
+2. listing 1.x, 1.y, or 1.z in milestones or in latest-milestone
+-->
+
+   - Stable
+     - [1209 - Metrics Stability](https://git.k8s.io/enhancements/...) - 1.21
+     - [1933 - Prevent logging secrets via static analysis](https://git.k8s.io/community/$link/README.md) - 1.23
+   - Beta
+     - [1602 - Structured Logging](https://git.k8s.io/community/$link/README.md) - 1.23
+     - [1748 - Pod resource requests/limits metrics](https://git.k8s.io/community/$link/README.md) - 1.22
+   - Alpha
+     - [2305 - Metrics Cardinality Enforcement](https://git.k8s.io/community/$link/README.md) - 1.21
+     - [647 - API Server Tracing](https://git.k8s.io/community/$link/README.md) - 1.22
+     - [2845 - Deprecate klog-specific flags in k8s components](https://git.k8s.io/community/$link/README.md) - 1.23
+   - Pre-alpha
+     - [2831 - Kubelet OpenTelemetry Tracing](https://git.k8s.io/community/$link/README.md) - alpha in 1.24
+
+## Project health
+
+1. What areas and/or subprojects does your group need the most help with?
+   Any areas with 2 or fewer OWNERs? (link to more details)
+
+   - [kubernetes-sigs/custom-metrics-apiserver](https://github.com/kubernetes-sigs/custom-metrics-apiserver/blob/master/OWNERS) has 1 active approver
```

Comment:
Remove custom-metrics-apiserver and metrics-server from this list.  I mistakenly considered someone inactive when they have actually been active.

Reply:
Based on https://github.com/kubernetes/community/pull/6463/files/83a275c21cb7143055f26b7a813e1c7cd628503c#r806269581, would it make sense to add klog?

Reply:
I don't think so, seriathius and pohly are pretty active on it.